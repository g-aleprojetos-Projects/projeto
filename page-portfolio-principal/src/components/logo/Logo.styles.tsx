import styled from 'styled-components';
import SVG from 'react-inlinesvg';

export const Container = styled.div`
  display: flex;
  align-items: center;
  justify-items: center;
`;

export const Iconelogo = styled(SVG)`
  display: flex;
  align-items: center;
  justify-content: center;
  width: 7rem;
  height: 7rem;
`;
