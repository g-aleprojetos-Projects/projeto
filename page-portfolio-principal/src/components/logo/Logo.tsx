import React from 'react';
import logo from 'assets/icon/logo.svg';
import * as S from './Logo.styles';

export const Logo = () => {
  return (
    <S.Container data-testid={'test_container_logo'}>
      <S.Iconelogo data-testid={'teste_logo'} src={logo} />
    </S.Container>
  );
};
