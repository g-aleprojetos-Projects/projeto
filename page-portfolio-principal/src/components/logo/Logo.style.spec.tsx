import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import 'jest-styled-components';
import logo from 'assets/icon/logo.svg';
import * as S from './Logo.styles';

describe('Logo.styles', () => {
  test('Container DEVE ser igual ao snapshot', () => {
    const container = renderer.create(<S.Container />).toJSON();
    expect(container).toMatchSnapshot();
  });
  test('Iconelogo DEVE ser igual ao snapshot', () => {
    const iconelogo = renderer.create(<S.Iconelogo src={logo} />).toJSON();
    expect(iconelogo).toMatchSnapshot();
  });
});
