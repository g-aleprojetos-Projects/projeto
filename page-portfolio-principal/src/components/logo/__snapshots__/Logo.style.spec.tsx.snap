// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo.styles Container DEVE ser igual ao snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  margin-top: 1.313rem;
}

<div
  className="c0"
/>
`;

exports[`Logo.styles Iconelogo DEVE ser igual ao snapshot 1`] = `null`;
